src/OWML.ModHelper.Menus/ModInputMenu.cs

```diff
@@ -60,6 +61,7 @@ public IModInputMenu Copy()
 			var newPopupObject = CopyMenu();
 			var newPopup = new ModInputMenu(Console);
 			newPopup.Initialize(newPopupObject.GetComponent<PopupInputMenu>());
+			newPopup._inputMenu.SetValue("_initialized", false);
```

Comment:
make a public method (and add to interface) for this?

Reply:
`_initialized` is from OW. I'm mostly just replicating it calling `Initialize`(maybe I should do it in Open though). Not sure if it's even necessary TBH

Reply:
🤷 👍